seed/tests/test_analysis_pipelines.py

```diff
@@ -508,7 +508,7 @@ def setUp(self):
                 # override unitted fields so that hashes are correct
                 site_eui=ureg.Quantity(
                     float(property_view_factory.fake.random_int(min=50, max=600)),
-                    "kilobtu / foot ** 2 / year"
+                    "kBtu / foot ** 2 / year"
```

Comment:
`kilobtu` is not a valid unit in the updated registry. These tests were the only locations I found that used the term

Reply:
nice.